docs/CONNECTOR.md

```diff
@@ -0,0 +1,51 @@
+# Connector toolkit
+This library includes a base toolkit to create a ship - connector to integrate Hull platform with a selected 3rd party API Service.
+
+## new Hull.App()
+```js
+import Hull from "hull";
+
+const app = new Hull.App({ port, hostSecret, service });
+
+// ... application configuration
+
+app.start();
+
+```
+
+### server()
+```js
+const app = new Hull.App({ port, hostSecret, service });
+
+const server = app.server();
```

Comment:
This entertains the confusion even more.
We are using `app` (which is still not an express app) and calling `server` to create an express app...
And then we're not calling it `app` but `server`, and we're not setting it up.

The reason we want to set it up:
- we want control over the port
- we might want to add middlewares before and after hull does it's job
- we want to be able to set the engines and static assets. Hull shouldn't force and own this. it's "good" boilerplate.

- we can't setup an express router here unfortunately since routers can't have engines. `router.set` isn't supported